### ISSUE TITLE
WiFi Sensors Missing

### ISSUE THREAD

**Issue by tescophil** [tchellomello/python-ring-doorbell#187]
After updating to the latest version, all WiFi sensors are missing from Doorbell and Chime entities. Previously there two WiFi sensors per device.

**Comment by brama1966** [comment]
Yep they have vanished and i cant see any errors in the log about it as i did before

**Comment by dshokouhi** [comment]
So these sensors were causing lots and lots of calls and IMO they are useless.  Ring devices are stationary and do not move (with the exception of the peephole cam that moves when the door opens/closes) so what use cases did you guys have?  Its either connected or not right?  Is this impacting any automation?  Note: I did not make this change but I did point out the fact that these devices are stationary and so this value should not be changing that much.
